### PR TITLE
List renderer template bs3 compability

### PR DESCRIPTION
**Pull request by bscheshirwork**
This is changes of ListRenderer for avoid table tag: #227

Include sortable widget; Include pass params to `prepareValue`
depend of #269

### CODE REVIEW

**Comment by bscheshirwork** [comment]
Note: This PR is changein existing html structure. Please check it as trigger for new release.

File: src/renderers/ListRenderer.php [hunk 1]
```diff
@@ -364,4 +384,27 @@ protected function prepareTemplate()
...
     }
+
+    /**
+     * Returns an array of JQuery sortable plugin options for ListRenderer
+     * @return array
+     */
+    protected function getJsSortableOptions()
+    {
+        return [
+            'containerSelector' => '.list-renderer',
```
Comment (unclead) [inline src/renderers/ListRenderer.php]: i think it is better to call parent method and just override `containerSelector`

**Comment by unclead** [comment]
@bscheshirwork did you check that everything works as expected?

**Comment by bscheshirwork** [comment]
I use ListRenderer instead original (throw composer autoloader classMap) for render MultipleInput in bootstrap3 theme. 

Look like everything works as expected... :smiley_cat:

File: src/renderers/ListRenderer.php (hunk 1 above)
Comment (bscheshirwork) [reply src/renderers/ListRenderer.php]: For table
```php
            'containerSelector' => 'table',
            'itemPath'          => '> tbody',
            'itemSelector'      => 'tr',
```
for list
```php
            'containerSelector' => '.list-renderer',
            'itemSelector' => '.multiple-input-list__item',
```

Comment (bscheshirwork) [reply src/renderers/ListRenderer.php]: So... I try with
```php
    protected function getJsSortableOptions()
    {
        return ArrayHelper::merge(parent::getJsSortableOptions(),
        [
            'containerSelector' => '.list-renderer',
            'itemPath' => new UnsetArrayValue,
            'itemSelector' => '.multiple-input-list__item',
        ]);
    }
```

Comment (bscheshirwork) [reply src/renderers/ListRenderer.php]: look like work fine

**Pull request merged**